### ISSUE TITLE
Feature request: Add init param `secure` to Storage to ignore CV code

### ISSUE THREAD

**Issue by dwhswenson** [openpathsampling/openpathsampling#693]
Would it be easy to add a `secure` parameter to storage initialization, which prevents CVs from also loading their code?

I imagine it functioning like this:

```python
new_snapshot = # snapshot not in filename.nc

st = paths.Storage("filename.nc", secure=False)  # default
cv = st.cvs[0]
cv(st.snapshots[0])
# returns value
cv(new_snapshot)
# returns value
st.close()

st = paths.Storage("filename.nc", secure=True)
cv = st.cvs[0]
cv(st.snapshots[0])
# returns value
cv(new_snapshot)
# raises error; maybe RuntimeError: No function associated with CV {cvname}.
#                                   It was probably loaded in secure mode.
#                                   You can manually associate a function with it with ...
```

This serves two valuable purposes:

1. **Security with untrusted data files.** This way, you should be able to load the saved results without having to worry about arbitrary executable code.
2. **Problems with Python 2 vs. Python 3 function representations.** It looks like we're having trouble reloading files generated in Python 2 when using Python 3. My guess is that the functions are encoded differently between the two; at least, the problem is occurring in the CV code reloading. This would give a quick workaround: load it as insecure, and then manually attach a function to the CV if you need to. (For pure analysis, you'll may only need the diskcached results.)

Thoughts, @jhprinz?

**Comment by jhprinz** [comment]
@dwhswenson This is done, right?

**Comment by dwhswenson** [comment]
Yep. Should have been closed by #695.
